ui/partials/perf/graphsctrl.html

```diff
@@ -60,8 +54,7 @@
       <div>
         <p id="tt-series"><span ng-bind="tooltipContent.test"/>
           (<span ng-bind="tooltipContent.project.name"/>)</p>
-        <p id="tt-series2" class="small"><span ng-bind="tooltipContent.platform"/>
-          (<span ng-bind="tooltipContent.machine"/>)</p>
+        <p id="tt-series2" class="small"><span ng-bind="tooltipContent.platform"/></p>
```

Comment:
machine was never defined, so this always wound up beig displayed as (mean), which is misleading. I opted to just take it out (technically I suppose I could/should file a seperate bug for this)